proj/src/components/statistics/statistics.js

```diff
@@ -24,8 +27,12 @@ export default class Statistics extends Component {
     ]
   }
 
+  componentDidMount() {
+    this.props.fetchData()
+  }
+
   drawBars(){
-    let data = this.state.data || [];
+    let data = this.props.data || [];
```

Comment:
i don't think you actually need these next 2 lines. data can be used from props and dataKeys can be initiated with the flow statement